### ISSUE TITLE
📢 Flyway 10.15.1 Maven Central Avaliblity

### ISSUE THREAD

**Issue by Barry-RG** [flyway/flyway#3916]
Unfortunately due to issues with deploying to maven central. The OSS artifacts for Flyway 10.15.1 are currently not available on Maven Central. We are currently looking at this issue but it is possible this release will be skipped on Maven Central and we will continue with the next release. 

The 10.15.1 source code is still available on this repository to build locally. 

Apologies for the incontinence.

**Comment by jmcruz1983** [comment]
10.15.1 is not available in docker hub either,
are you skipping these builds as well?

**Comment by Barry-RG** [comment]
@jmcruz1983 yes unfortunately, our Docker hub release required the maven central deployment. Apologies.
